core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/ModeResource.java

```diff
@@ -163,6 +165,7 @@ public Mode getMode(
   }
 
   @PUT
+  @DocumentedName("updateMode")
```

Comment:
Will `updateGlobalMode` be a better name here?

Reply:
Addressed